src/domain/prices/prices.repository.ts

```diff
@@ -16,21 +16,29 @@ export class PricesRepository implements IPricesRepository {
     nativeCoinId: string;
     fiatCode: string;
   }): Promise<number> {
-    const result = await this.coingeckoApi.getNativeCoinPrice(args);
+    const lowerCaseFiatCode = args.fiatCode.toLowerCase();
+    const result = await this.coingeckoApi.getNativeCoinPrice({
+      nativeCoinId: args.nativeCoinId,
+      fiatCode: lowerCaseFiatCode,
+    });
     const assetPrice = await this.assetPriceValidator.validate(result);
```

Comment:
No need for `await` here.

Reply:
Changed in [b69541f](https://github.com/safe-global/safe-client-gateway/pull/768/commits/b69541f62eedaf3a1a759dab61d4a1ad90206006)